src/validator.js

```diff
@@ -84,6 +100,59 @@ export default class Validator {
     return JSON.stringify.apply(null, args);
   }
 
+  textOutput() {
+    var out = [];
+
+    out.push(_('Validation Summary:'));
+    out.push('');
+    out.push(columnify(this.output.summary, {
+      showHeaders: false,
+      minWidth: 15,
+    }));
+    out.push('');
+
+    for (let type of constants.MESSAGE_TYPES) {
+      /*eslint-disable no-loop-func */
```

Comment:
We're using `let` so this shouldn't be an issue: http://eslint.org/docs/rules/no-loop-func.html#rule-details

Was this giving you an error? :frowning: